packages/sdk/src/utils.ts

```diff
@@ -25,6 +26,7 @@ export const getCurrentTimestamp = (delay = 0) =>
 export const lc = (x: string) => x?.toLowerCase();
 export const n = (x: any) => (x ? Number(x) : x);
 export const s = (x: any) => (x ? String(x) : x);
+export const toCheckSum = (x: any) => utils.getAddress(x);
```

Comment:
For consistency with the rest of the code, we should always lowercase addresses instead of storing them checksummed.